src/components/header/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext, useEffect } from 'react';
+import './header.scss';
+import { Navigation } from '../navigation/Navigation';
+import { GlobalContext } from '../../reducer';
+import { requestListProducts } from '../../helpers/api';
+
+export const Header = () => {
+  const [state, dispatch] = useContext(GlobalContext);
+
+  useEffect(() => {
+    dispatch({ type: 'load', active: true });
+    requestListProducts()
```

Comment:
would be better to use `async/await` syntax;
it's important when your portfolio project will check some reviewers

Reply:
+